core/src/sumeragi/main_loop.rs

```diff
@@ -13,17 +10,7 @@ use tracing::{span, Level};
 use super::*;
 use crate::{block::*, sumeragi::tracing::instrument};
 
-/// Fault injection for consensus tests
-pub trait FaultInjection: Send + Sync + Sized + 'static {}
```

Comment:
Great to know that it could be deleted that easily. Non-generic signatures are much easier to read.